src/main.cpp

```diff
@@ -352,6 +311,18 @@ MODLOADER_FUNC void modloader_preload(JNIEnv* env, char const* appId, char const
     LOG_FATAL("Failed to copy over files! Modloading cannot continue!");
     failed = true;
   }
+  if(runtime_restriction::init(std::filesystem::path(modloaderSource).filename().string())) {
+    std::vector<std::string> ld_paths = { "/vendor/lib64", "/system/lib64", "/system/product/lib64" };
```

Comment:
Would we want more than this? What dictates these strings being present?

Reply:
These are system lib paths idk of any other default paths

Reply:
there's the non 64 variants of these as well, and I assume the actual game directory would be one as well (where libil2cpp and libunity are)

Reply:
If you have the non 64 variants it will conflict for the same filename being in both folders
The game directory is game dependent so I'd like to not use some hardcoded string for that